cmake/compiler/arcmwdt/compiler_flags.cmake

```diff
@@ -132,6 +132,9 @@ set_property(TARGET compiler-cpp PROPERTY dialect_cpp2a "")
 set_property(TARGET compiler-cpp PROPERTY dialect_cpp20 "")
 set_property(TARGET compiler-cpp PROPERTY dialect_cpp2b "")
 
+# Flag for disabling strict aliasing rule in C and C++
+set_compiler_property(PROPERTY no_strict_aliasing -fno-strict-aliasing)
```

Comment:
@evgeniy-paltsev @abrodkin can you please check if this works on arcmwdt?

Reply:
Works perfectly fine for ARC MWDT, thanks!